ironfish/src/telemetry/telemetry.ts

```diff
@@ -131,12 +164,12 @@ export class Telemetry {
         {
           name: 'inbound_traffic',
           type: 'float',
-          value: this.metrics.p2p_InboundTraffic.rate1s,
```

Comment:
I'm not sure why this was `rate1s` before but it seems to make more sense to send the 5m rolling average if we are sending stats every 5 minutes

Reply:
Seems fine to change